composer.json

```diff
@@ -67,7 +67,7 @@
         "npm-asset/jquery-ui-touch-punch": "^0.2.3",
         "oomphinc/composer-installers-extender": "^2.0",
         "php": ">=8.0",
-        "rhodeislandecms/ecms_profile": "0.9.16",
+        "rhodeislandecms/ecms_profile": "dev-RIGA-329/update-acquia-connector-module",
```

Comment:
Use `dev-staging` here

Reply:
If you have changes in `ecms_profile` you need to test this, merge those to that staging branch as well